docs/reports/requirements/2_dfd-desc.md

```diff
@@ -1,25 +1,20 @@
 
-## Level 0 DFD 
-
-![](dfd_0.png)
+## Level 0 
 
+The level 0 data flow diagrams give a high level view of how our system will be interacting with the user and the client’s existing system. 
 
+![](dfd_0.png)
 
+The user will first access Kukurella’s text-to-speech front end and type in a text input. The user will also select a voice and an avatar they would like to use for their text-speech animation. From the front-end the text and other user selections will be sent over to our Animation Add-On. The Animator will then query an avatar image, avatar coordinates, and mouth animation assets from a database. The Animator will produce an animation of an avatar reading the user’s text in the user selected voice. This animation will be sent to Kukeralla's web application and then will be sent to the user.  
 
-- Four main groups listed in this DFD: user, kukurella Web App, Animation Process, and Assets library
-- The web app is already in place, will be creating an animation add-on that receives inputs from this web app
-- Assets library will include avatar images mouth assets, and coordinates to indicate where to place mouth assets onto the avatar 
+## Level 1
 
-## Level 1 DFD
+The level 1 DFD provides a more detailed breakdown of the data flow between the client’s system and the Animation Add-On.
 
 ![](dfd_1.png)
 
-- The kukurella web app can be divided into a web app and an api endpoint that we will be touching
-- The API will provide User text, language, and audio for our phoneme generation process
-- The phoneme factory will provide the animation processor with a phoneme text file
-- The animation processor will map phonemes with mouth assets found in the assets directory
-- The processor will then sequence these mouth assets over a 2d animation avatar
-- The output will be an animation file that will be sent back to the front-end
+In this diagram Kukeralla’s Web application is broken down into a front-end and a text-to-speech API. The Kurkarella text-to-speech API creates the synthesized speech file from the user’s voice and text input. The Animation add-on is broken down into two main components. The phoneme factory is in charge of mapping spoken language from an audio file into a series of phonemes. In order to make this process as accurate as possible the phoneme factory also receives the user text file and language selection. Once the phoneme file is created it is sent off to a secondary process called the phoneme processor. This processor maps phonemes from the phoneme file to mouth assets received from a directory. There will be 1:1 mapping between phonemes and mouth assets. These assets will be added onto the avatar and rendered into a video file. The video file will be sent back to the front-end and available for the user to download onto their local machine.
```

Comment:
Suggestions:
- in the project description, I didn't capitalize the W in "web application." We should be consistent, so if we should capitalize it, let me know and I'll adjust my section!
- please be consistent with capitalizing "Animation Add-on" or "Animation add-on"
- it might be a good idea to try and define what a phoneme is - technically, the Rhubarb tool is extracting the phonemes for English text and speech, but for non-English, it's mapping the sounds to the closest phoneme, but we don't really need to make that distinction in here. In the intro, I just said it synchronized the lips to the sounds, but it's nice that you're going more in-depth here!
- If you are capitalizing the Animator, it might be worth capitalizing the Processor. Alternatively, don't capitalize them... As long as you're consistent, it's fine!